xinference/web/ui/package-lock.json

```diff
@@ -27,7 +27,6 @@
         "@testing-library/react": "^13.4.0",
```

Comment:
The changes in this file should be reverted.

Reply:
Fixed.